main.rb

```diff
@@ -0,0 +1,52 @@
+require_relative 'app'
+
+def options
+  puts ''
+  puts 'Welcome to School Library App!'
+  puts ''
+
+  puts '1 - List all books',
+       '2 - List all people',
+       '3 - Create a person',
+       '4 - Create a book',
+       '5 - Create a rental',
+       '6 - List all of rentals for a given person id',
+       '7 - Exit'
+  puts ''
+  puts 'Please choose an option by entering a number: '
+  print '#=> '
+end
+
+def select_option(app, num)
+  case num
+  when '1'
+    app.list_all_books
+  when '2'
+    app.list_all_people
+  when '3'
+    app.create_person
+  when '4'
+    app.create_book
+  when '5'
+    app.create_rental
+  when '6'
+    app.list_rentals
+  else
+    puts ''
+    puts '#=> Invalid option.(InvalidInputError)'
+  end
+end
+
+def main
+  app = App.new
+  loop do
+    options
+    num = gets.chomp
+    break if num == '7'
+
+    select_option(app, num)
+    puts "\n"
+  end
+
+  puts 'Thanks for using the app. Goodbye'
+end
```

Comment:
- [ ] The `ruby main.rb` console command fails to run your application. This is because the `main` method is not being called at the bottom of this main.rb file. Please try to make it work 👍 
- [ ] After you have fixed this issue, the next reviewer will check to see if you have satisfied all the requirements to proceed to the next project.
![image](https://github.com/aima98/OOP-School-Library/assets/46839548/91462c5c-1002-4eda-a5df-9b9dfd4f010e)